<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="apiGroupBox">
     <property name="title">
      <string>APIs Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QListWidget" name="apisListWidget">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="apiKeyGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="apiKeyLabel">
            <property name="text">
             <string>API key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="apiKeyLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="textTypeGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textTypeLabel">
            <property name="text">
             <string>Text type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="textTypeLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="defaultSourceLanguageGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="defaultSourceLanguageLabel">
            <property name="text">
             <string>Default source language</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="defaultSourceLanguageLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="defaultTargetLanguagegridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="defaultTargetLanguageLabel">
            <property name="text">
             <string>Default target language</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="defaultTargetLanguageLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="defaultApiLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="text">
      <string>Default API</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="settingsButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="defaultApiComboBox"/>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>settingsButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
